packages/discovery-react-components/src/components/DocumentPreview/components/PdfViewer/PdfViewer.tsx

```diff
@@ -56,7 +56,11 @@ export type PdfViewerProps = PdfDisplayProps & {
    */
   setRenderedText?: (info: PdfRenderedText | null) => any;
   /**
-   * URL of hosted PDF worker
+   * Callback any errors on render
+   */
+  setIsPdfRenderError: (isError: boolean) => void;
+  /**
+   * Callback for text layer info
```

Comment:
Why did you change the description of this variable? Shouldn't it still be `URL of hosted PDF worker`?

Reply:
Thanks for catching my copy-and-paste. I restored the original description.